src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
since you are raising the error, do you wish for the script to stop mid flow?
if yes, then what you have is sufficient, if no then the lower ranked version (what I recommend to do) is better.